var/spack/repos/builtin/packages/subversion/package.py

```diff
@@ -67,9 +67,9 @@ def install(self, spec, prefix):
         options.append('--with-sqlite=%s' % spec['sqlite'].prefix)
         options.append('--with-serf=%s' % spec['serf'].prefix)
 
-        if spec.satisfies('^swig'):
+        if 'swig' in spec:
             options.append('--with-swig=%s' % spec['swig'].prefix)
-        if spec.satisfies('+perl'):
```

Comment:
Did this fail as well?  This tests the variant and is actually a different check from `^swig`.